lib/galaxy_test/api/test_licenses.py

```diff
@@ -0,0 +1,23 @@
+from ._framework import ApiTestCase
+
+
+class LicensesApiTestCase(ApiTestCase):
+
+    def test_index(self):
+        response = self._get("licenses")
+        self._assert_status_code_is(response, 200)
+
+    def test_get_license(self):
+        licenseId = "Apache-2.0"
+        response = self._get(f"licenses/{licenseId}")
+        self._assert_status_code_is(response, 200)
+        self._assert_matches_license_id(licenseId, response.json())
+
+    def test_404_on_unknown_license(self):
+        licenseId = "uknown"
+        response = self._get(f"licenses/{licenseId}")
+        self._assert_status_code_is(response, 404)
```

Comment:
Haha, so you found a bug in the old API route, which returns a 200 here. Can you use `get_license_by_id` in the old endpoint too ?

Reply:
Sure! :)